ui/app/helpers/icon-for-component.ts

```diff
@@ -14,7 +14,7 @@ export function iconForComponent([component]: [string]): string {
     case 'google-cloud-run':
       return 'logo-gcp-color';
     case 'kubernetes':
-      return 'logo-kubernetes-color';
+      return 'logo-kubernetes-color-alt';
```

Comment:
I think If you upgrade structure-icons to the latest you won't need this, but that can be a follow up.

Reply:
Yes, thank you!